ee/desktop/user/notify/notify_darwin.go

```diff
@@ -46,6 +42,12 @@ func (m *macNotifier) Interrupt(err error) {
 	m.interrupt <- struct{}{}
 }
 
+func (m *macNotifier) Listen() {
```

Comment:
Maybe a quick comment here about what you found/why this is needed?

Reply:
done